tests/integration_tests/test_hyperopt_ray_horovod.py

```diff
@@ -81,25 +81,25 @@ def mock_storage_client(path):
             "dynamic_resource_allocation": True,
         },
     },
-    {
-        "type": "ray",
-        "search_alg": {"type": "bohb"},
-        "scheduler": {
-            "type": "hb_bohb",
-            "time_attr": "training_iteration",
-            "reduction_factor": 4,
-        },
-        "num_samples": 3,
-    },
+    # {
```

Comment:
Why are these tests commented out? If we want to leave them disabled like this for now, could you add a TODO with a reference to an issue to re-enable them?

Reply:
Updated in the latest commit.